mobify/progressive-web-scaffold
# [Icons] Update CC SVG assets

## Changes
- Update SVG assets that are slightly smaller in file size
- Removed unused `cc-card.svg` file

## How to test-drive this PR
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Navigate to: [Payment step](https://www.merlinspotions.com/checkout/payment/)
- Try different CC numbers to confirm SVG still loads
